testapps/svelte/src/i18n/cs.json

```diff
@@ -1,12 +1,12 @@
 {
-  "add-item-add-button" : "Přidat",
-  "add-item-input-placeholder" : "Nová položka seznamu",
-  "delete-item-button" : "Smazat",
-  "menu-item-translation-methods" : "Metody překládání",
-  "on-the-road-subtitle" : "Co si sbalit na výlet",
-  "on-the-road-title" : "Na cestu",
-  "send-via-email" : "Odeslat e-mailem",
-  "share-button" : "Sdílet",
-  "this_is_a_key" : "Toto je klíč",
-  "this_is_a_key_with_params" : "Toto je klíč s parametry {key} {key2}"
-}
\ No newline at end of file
+  "add-item-add-button": "Přidat",
```

Comment:
The translation files are generated by Tolgee platform, so the prettier should ignore them

Reply:
fixed